rendercv/data/models/render.py

```diff
@@ -0,0 +1,131 @@
+"""
+The `rendercv.models.render` module contains the data model of the
+`rendercv_settings.render` field of the input file.
+"""
+
+from typing import Optional
+
+import pydantic
+
+from .base import RenderCVBaseModelWithExtraKeys
+
+
+class RenderOptions(RenderCVBaseModelWithExtraKeys):
```

Comment:
You can move all this to `rendercv_settings`. We don't need a seperate file for now.

Reply:
I would argue here. I would immediately start working on #164 and this implementation would be more helpful for that task

Reply:
There are too many files now. We might consider folders inside the `models` folder (but maybe in another PR).

Reply:
Okay.